src/test/java/com/google/sps/servlets/AddClassTATest.java

```diff
@@ -0,0 +1,283 @@
+  public void addOneTAEmptyList() throws Exception {
+
+    // Create a user
+    Entity user = new Entity("User");
+
+    user.setProperty("userEmail", "test@google.com");
+    user.setProperty("registeredClasses", Collections.emptyList());
+    user.setProperty("taClasses", Collections.emptyList());
+    user.setProperty("ownedClasses", Collections.emptyList());
+
+    datastore.put(user);
+
+    // Create examples for the TA email and class code
+    when(httpRequest.getParameter("taEmail")).thenReturn("test@google.com");
+    when(httpRequest.getParameter("classCode")).thenReturn("exampleClassCode");
+
+    addTA.doPost(httpRequest, httpResponse);
+
+    Query query = new Query("User");
+    PreparedQuery results = datastore.prepare(query);
+
+    // Add the class key to the user's TA classes list
+    for (Entity entity : results.asIterable()) {
+      if (entity.getProperty("userEmail") == "test@google.com") {
+        List<Key> taClasses = (List<Key>) entity.getProperty("taClasses");
+        assertTrue(taClasses.contains(KeyFactory.stringToKey("exampleClassCode")));
+        assertTrue(taClasses.size() == 1);
+      }
+    }
+  }
+
+  @Test
+  // For a user that already TAs for one class, add more classes
+  public void addOneTANonEmptyList() throws Exception {
+
+    // Create a class
+    Entity init = new Entity("Class");
+
+    init.setProperty("owner", "ownerID");
+    init.setProperty("name", "testClass");
+    init.setProperty("beingHelped", new EmbeddedEntity());
+    init.setProperty("studentQueue", Collections.emptyList());
+    init.setProperty("taList", Collections.emptyList());
+
+    datastore.put(init);
+
+    // Initialize a user
+    Entity user = new Entity("User");
+
+    List<Key> taClassList = Arrays.asList(init.getKey());
+
+    user.setProperty("userEmail", "test@google.com");
+    user.setProperty("registeredClasses", Collections.emptyList());
+    user.setProperty("taClasses", taClassList);
+    user.setProperty("ownedClasses", Collections.emptyList());
+
+    datastore.put(user);
+
+    // Create examples for the TA email and class code
+    when(httpRequest.getParameter("taEmail")).thenReturn("test@google.com");
+    when(httpRequest.getParameter("classCode")).thenReturn("exampleClassCode");
+
+    addTA.doPost(httpRequest, httpResponse);
+
+    Query query = new Query("User");
+    PreparedQuery results = datastore.prepare(query);
+
+    // Add the class key to the user's TA classes list
+    for (Entity entity : results.asIterable()) {
+      if (entity.getProperty("userEmail") == "test@google.com") {
+        List<Key> taClasses = (List<Key>) entity.getProperty("taClasses");
+        assertTrue(taClasses.contains(KeyFactory.stringToKey("exampleClassCode")));
+        assertTrue(taClasses.contains(init.getKey()));
+        assertTrue(taClasses.size() == 2);
+      }
+    }
+  }
+
+  @Test
+  // For a user that already TAs for one class, add more classes
```

Comment:
This comment is the same as the previous test, can you update it to show the difference b/w the two tests